packer/ansible/roles/aggregate/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: install packages
+  apt:
+    pkg: '{{ item }}'
+    install_recommends: no
+  with_items:
+    - unzip
+
+- name: set facts
+  set_fact:
+    tomcat_webapps_dir: /var/lib/tomcat8/webapps
+    motd_dir: /etc/update-motd.d
+
+- name: setup database - create user odk
```

Comment:
I'd prefer we use aggregate as the username and the database. This makes it consistent with the older VMs and I think it's clearer than `odk`.

My gut also says we should source a single sql file to create the DB. If we can get it from the source, it'd be good so we don't have to change this later.

Reply:
I'm no Ansible expert and I didn't find a way to put an inlined `<<< EOF` like we do with the dev docker database. Then I saw some guides that recommended to place each SQL sentence separately.

How would you do this? Transfer a file to the VM, run it and then remove it? Any other way?

Reply:
Yeah, I usually put it in files/, then copy it over to /tmp on the VM, then run it. No need to remove because it will go away during packaging.

If you have links to those guides, please share. What you found might be the best practice!